applications/StructuralMechanicsApplication/tests/test_StructuralMechanicsApplication.py

```diff
@@ -20,68 +20,81 @@
 # Simple patch tests
```

Comment:
Is it ok for us to add this test later? We don't really know how these tests work yet and have to figure it out first. And besides there are test for every situation added.

There is anyway another bunch of stuff to be added coming soon, then we can also add tests that are in the new format